bbb/services.py

```diff
@@ -285,6 +285,16 @@ def reflectTasks(self):
             # If takenUntil isn't set, this task has either never been claimed
             # or got cancelled.
             if not t.takenUntil:
+                # If the buildrequest is showing complete, there is a
+                # possibility, that the build was completed before takenUntil
+                # was updated by BBListener. To avoid this we can try to avoid
+                # processing the buildrequest for 5 minutes.
+                if arrow.now() < arrow.get(t.processedDate).replace(minutes=5):
+                    log.debug(
```

Comment:
this probably won't happen often. I wonder if this should be an info/warn so we see it once we disable debug log lines